sygnal/apnstruncate.py

```diff
@@ -85,7 +87,7 @@ def truncate(payload: Dict[str, Any], max_length: int = 2048) -> Dict[str, Any]:
     return payload
 
 
-def _choppables_for_aps(aps):
+def _choppables_for_aps(aps: dict) -> List[Union[Tuple[str], Tuple[str, int]]]:
```

Comment:
I would probably narrow this a little
```suggestion
def _choppables_for_aps(aps: Dict[str, Any]) -> List[Union[Tuple[str], Tuple[str, int]]]:
```